contracts/contracts/strategies/NativeStaking/NativeStakingSSVStrategy.sol

```diff
@@ -113,6 +114,14 @@ contract NativeStakingSSVStrategy is
             _assets,
             _pTokens
         );
+
+        // Approves the SSV Network contract to transfer SSV tokens for deposits
+        IERC20(SSV_TOKEN).approve(SSV_NETWORK, type(uint256).max);
+
+        // Set the FeeAccumulator as the address for SSV validators to send MEV rewards to
+        ISSVNetwork(SSV_NETWORK).setFeeRecipientAddress(
```

Comment:
nit: not a big deal, but this could also be called from the constructor

Reply:
it can't as it needs to come from the proxy contract, not the implementation contract. Hence it's in the `initialize` function.